subsys/spm/spm.c

```diff
@@ -29,6 +29,13 @@
 #define NON_SECURE_APP_ADDRESS DT_FLASH_AREA_IMAGE_0_NONSECURE_OFFSET_0
 #endif /* USE_PARTITION_MANAGER */
 
+#if defined(CONFIG_SPM_DEBUG)
```

Comment:
Can't be used stand alone since it is not defined if `CONFIG_SPM_DEBUG=n`

Reply:
Not sure what you mean here, but if you meant to say that `#if CONFIG_SPM_DEBUG` would not have worked that's not the case since non-defined macros resolve to 0.